.github/workflows/unit-tests.yml

```diff
@@ -154,22 +154,36 @@ jobs:
       with:
         environment-file: ci/deps/${{ matrix.env_file }}
 
+    - name: Set sanitizer flags
+      run: |
+        echo "CFLAGS=$CFLAGS -fno-sanitize-recover=all" >> "$GITHUB_ENV"
```

Comment:
Having trouble setting this with GHA. There may also be a way to pass the flag directly through meson python? @lithomas1 any idea?